services/openshift/scripts/openshift

```diff
@@ -58,7 +58,7 @@ start_openshift() {
         -v ${ORIGIN_DIR}/openshift.local.volumes:${ORIGIN_DIR}/openshift.local.volumes:z \
         -v ${ORIGIN_DIR}/openshift.local.config:${ORIGIN_DIR}/openshift.local.config:z \
         -v ${ORIGIN_DIR}/openshift.local.etcd:${ORIGIN_DIR}/openshift.local.etcd:z \
-        openshift start "${@:2}"
+        $IMAGE start "${@:2}"
```

Comment:
I rearranged this to use `docker cp` after the final start.  This way we are more efficient